azure-storage-blob/azure/storage/blob/blob_service_client.py

```diff
@@ -396,14 +439,12 @@ def get_blob_client(
         :param blob: The blob with which to interact. If specified, this value will override
          a blob value specified in the blob URL.
         :type blob: str or ~azure.storage.blob.models.BlobProperties
-        :param ~azure.storage.blob.common.BlobType blob_type: The type of Blob. Default
-         vale is BlobType.BlockBlob
         :param str snapshot: The optional blob snapshot on which to operate.
         :returns: A BlobClient.
         :rtype: ~azure.core.blob.blob_client.BlobClient
         """
         return BlobClient(
-            self.url, container=container, blob=blob, blob_type=blob_type, snapshot=snapshot,
+            self.url, container=container_name, blob=blob_name, snapshot=snapshot,
```

Comment:
Are we intended to use **_container_name_** and **blob_name**? It seems **BlobClient** takes blob/container instance as a parameter instead of only the name.

Reply:
Thanks - yeah I've been a bit inconsistent here. I will consolidate the naming to use "container" and "blob" throughout